src/pages/SectionPage.jsx

```diff
@@ -60,14 +69,25 @@ const sectionsData = [
   },
 ];
 
+function CreateNewSectionSegment(onAddSegment, addSection) {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'CreateNewSectionSegment' is defined but never used.